scripts/frontend.sh

```diff
@@ -0,0 +1,5 @@
+set -euxo pipefail
+
+cd frontend
+npm install
```

Comment:
```suggestion
npm ci
```

Reply:
npm install runs a lot faster - typically it's fine for most use cases

Reply:
it's just that npm install might modify package.json
TBF we can spot that in PRs, let it be npm install